README.md

```diff
@@ -48,7 +50,7 @@ If you have Jupyter installed, you can create a notebook using IRkernel from the
 You can also start other interfaces with an R kernel:
 
 ```bash
-# “ir” is the kernel name installed by the above 'IRkernel::installspec()'
```

Comment:
Sorry, I've so many C&P problems due to non-ascii chars in code examples, I'm firmly in the camp of not writing any unicode chars if I don't have to...

Yes, I'm on windows...